src/main.ts

```diff
@@ -5,7 +5,7 @@ import { AppModule } from './app.module';
 async function startServer(): Promise<void> {
   const PORT = process.env.PORT || 3000;
   const app = await NestFactory.create(AppModule);
-  app.useGlobalPipes(new ValidationPipe());
+  app.useGlobalPipes(new ValidationPipe({ skipMissingProperties: true }));
```

Comment:
Why are the reasons for skipping missing properties in class-validator globally?

You are forced to apply `@IsDefined()` decorator for every input property, to keep validation work.

Reply:
This is needed when you create a patient, passing userId instead of creating new, in this case, class-validator throw an exception that says that create user dto fields are empty. So I'm using this option to pass through that error.
@IsDefined() I was using it because I didn't understand how it works and left it as it is. If I understand it right it is needed to validate the field when it has any value or not.